.github/workflows/i18n-string-sync-master.yml

```
@@ -0,0 +1,94 @@
+      - name: Set up environment
+        uses: ./actual/.github/actions/setup
+        with:
+          working-directory: actual
+      - name: Configure Git config
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+      - name: Check mergefreeze status
+        id: mergefreeze
+        shell: bash  # Enable pipefail
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:13: Quote this to prevent word splitting

(shellcheck)